text/0000-rfc-process-improvements.md

```diff
@@ -0,0 +1,72 @@
+# Meta
+[meta]: #meta
+- Name: RFC process improvements
+- Start Date: 2019-12-20
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: Amends [rfc#4](https://github.com/buildpacks/rfcs/pull/7/files)
+
+# Summary
+[summary]: #summary
+
+The RFC process provides the main vehical for substantial changes to the project, but needs amending to prevent RFCs getting stuck.
+
+# Motivation
+[motivation]: #motivation
+
+The RFC process currently has at least two problems that allow proposals to stall:
+1. It has an unclear forcing function (its unclear how to intiate the FCP process) and so RFCs can languish
+2. When voting members are absent RFCs cannot move forward
```

Comment:
I'd prefer not to address #2 as part of this RFC.  I believe that the better way to solve this is by elevating additional core team members, which in practice will create redundant votes for component companies.  This gives the needed redundancy when people are away.

Reply:
Sounds reasonable. I'll take that aspect out.